x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/FollowerFailOverIT.java

```diff
@@ -164,6 +165,12 @@ public void testFollowIndexAndCloseNode() throws Exception {
         int maxOpsPerRead = followRequest.getParameters().getMaxReadRequestOperationCount();
```

Comment:
I haven't really figured out how this test works yet, but there's an interesting [putFollow() method variation](https://github.com/elastic/elasticsearch/blob/e77dc6dc1324b00ba2ddf15718686e8bd1bcb3a9/x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/CcrIntegTestCase.java#L590) that seems to do something similar to your change

Reply:
Hmm we're actually calling that method with `ActiveShardCount.ONE` already, so you make a good point. Why are we not already waiting for these shards to be active?

Reply:
Oops, I updated the comment and probably raced. What about using `ActiveShardCount.ALL`? We've [got 3 shards](https://github.com/elastic/elasticsearch/blob/e77dc6dc1324b00ba2ddf15718686e8bd1bcb3a9/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/FollowerFailOverIT.java#L126), not 1.

Reply:
`ActiveShardCount#ALL` is right, but not because there are 3 shards. This parameter applies to each replication group separately, so `ActiveShardCount#ONE` means "wait for all three primaries to be assigned". But in this case we're shutting down one of the nodes, so we need the replicas to be up and running too.